cpp/tests/types/types_test.cpp

```diff
@@ -283,7 +283,7 @@ TEST_F(WrappersTestBool8, BooleanOperatorsBool8)
 TEST_F(WrappersTestBool8, CastArithmeticTest)
 {
     cudf::bool8 w1{42};
-    cudf::bool8 w2{static_cast<char>(-42)};
+    cudf::bool8 w2{-42};
```

Comment:
Why did you change this back? This will cause compilation issues on ARM.

Reply:
Great catch. I don't think I touched this but it's likely I borked something when resolving a merge conflict